src/utils/nanvixd/src/control_plane.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright(c) The Maintainers of Nanvix.
+// Licensed under the MIT License.
+//
+// This file contains the messages used in the control-plane API between
+// nanvixd and linuxd/user VM.
+
+use ::anyhow::Result;
+use ::std::io::{
+    Error,
+    ErrorKind,
+};
+use ::syscomm::{
+    SocketError,
+    SocketStream,
+};
+
+#[repr(u8)]
+#[derive(Clone, Copy, Debug)]
+pub enum Command {
+    Shutdown = 1,
```

Comment:
Can we have a global constant for this and use it throughout the file?

Reply:
I am not sure I understand. `Shutdown` is one of the possible commands we will have. I am giving it a number because to avoid implementing an expensive protocul, I used `[repr(u8)]` to have rust automatically (de-)serialize it for us.

Reply:
Yes, still you are hardcoding the constant value when handling the conversion. You could alternatively use num_enum crate.

Reply:
Makes sense. Done.